README.md

```diff
@@ -6,94 +6,68 @@ drandjs is a Javascript library able to communicate with a public [drand](https:
 
 ## Installation
 
-There are two ways to install drandjs: by using a CDN or compile it locally and bundle it with your application.
-
-In the former case, you can use:
+The simplest way to use drandjs is to import the script from *jsdelivr* as in the following:
 ```javascript
 <script src="https://cdn.jsdelivr.net/gh/PizzaWhisperer/drandjs/dist/drand.js"></script>
 ```
-In the latter case, simply run `make compile` to bundle every file together and create the `dist/drand.js` file ready to use in your application.
-
-## Usage
 
-drandjs has several public functions which allow to fetch configuration files or *public* randomness from a drand node. In order to know which drand node to contact and to be able to verify the randomness fetched from this node, an identity and a distributed key have to be specified.
+**Compile from sources**: To compile the sources into a single bundled Javascript file, you can run `make compile` to create the file `dist/drand.js` ready to be included in your application.
 
-### Identity
+## Usage
 
-The identity structure has the following form:
-```json
-{
+The following is a Javascript snippet showing the shortest way to get verified randomness from a running drand network:
+```javascript
+// the identity of the node we want to contact
+var identity = {
   Address: "drand-test.nikkolasg.xyz:8888",
   TLS: true,
-}
-```
-The **Address** is the IP address or DNS name of the drand node the user wishes to contact. **TLS** is true if drandjs should contact this node over HTTPS (drand nodes by default are using HTTPS) and false if the node does not have a TLS certificate.
-One can retrieve the identity of drand nodes from the group configuration file of the network. See [drand](https://github.com/dedis/drand) for more information.
+};
+// the distributed key of the public drand network (optional).
+var dist_key = "30572b6ff54ab9b0a17ed055ffb915a9a31640ee1a10ed20ace8a2394d1121bb3fe4424a24323566b9ed3b6d4aaf43ef9351e1c989fd5e194e27c3d2ef4014586aee32472dddafb6f28f5b36fefdb7863f31f897684e203cd05ad5486baf602f84f02570f7385bc360a577111f5b03387c00d548cc2276a19b3c2e317117baba";
 
-### Distributed Key
+fetchAndVerify(identity, dist_key)
```

Comment:
I would put the snippet after the API, we are introducing a func without explaining what it does

Reply:
Mhhh I tend to disagree lol but that's my own opinion. Showing a quck example of how a library works is very usual on Github in my opinion.A middleground option could be to add a short explanation over the method to explain directly in the snippet ?

Reply:
If you think it's the right way to do it then let's leave it like this:)

Reply:
I'm fine to ask Philipp what he thinks of it as well if you wish - you can defend your own opinion ;)